src/tests/base/test_payment.py

```diff
@@ -1,5 +1,6 @@
 from decimal import Decimal
 
+import datetime
```

Comment:
As `now` is already imported in this file, use it here, too, please. 

(But just for reference, the style tests were failing because the datetime import is in the wong line - don't worry about that, though. In the future, you might want to run `isort -rc .` before committing if you add new imports. Just an aside, don't worry about it.)